NewForce-Cohort-6/tabloidfullstack-stuffing
# SB/6-Create-a-Post

# Description

Functionality added for a user to add a new post.

# Testing Instructions

`git checkout SB/6-Create-a-Post`

1. Login with any user
2. Navigate to New Post via navbar
3. You should see a form with Title, Content, Category, Header Image URL and Published Date fields
4. Title, Content and Category should be required. Header Image URL and Published Date can be left empty.
5. Cancel should bring you back to the home screen.
6. Save should post the new post to the database and redirect you to the new post's post details page.

# Checklist:
